### PR TITLE
[WIP] Blog: learning Nextflow in 2022

### PR DESCRIPTION
**Pull request by evanfloden**
Ready for building an a final check

### CODE REVIEW

File: content/blog/2022/learn-nextflow-in-2022.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+title=Learn Nextflow in 2022
+date=2022-01-21
+type=post
+description=Resources for learning Nextflow: 2022 Revision
```
Comment (pditommaso) [inline content/blog/2022/learn-nextflow-in-2022.md:4]: It was nicer "Learning Nextflow in 2022"! 😄

Comment (evanfloden) [reply content/blog/2022/learn-nextflow-in-2022.md:4]: Updated.

**Comment by evanfloden** [comment]
Marked as WIP, awaiting an updated social share image.

**Review by pditommaso — APPROVED** [review]

**Comment by pditommaso** [comment]
ouch, didn't notice last comment, but it looks image was uploaded

**Pull request merged**